### PR TITLE
Splits off Exploration and Science

### PR DESCRIPTION
**Pull request by Novacat**
- Removes all Science access from explorers
- Removes Gateway access from HoS and RD
- Removes Science comms from explorer headsets
- Adds Command comms to Pathfinder headset
- Refactors edited Southern Cross files into _vr files in the main code.
- Moves Exploration Equipment Vendor to the expedition shuttle.
- Moves the Chemical Analyzer in science to Exploration Prep
- Removes general science access from the Xenobiology office and equipment rooms
- Removes RD access to Explorer comms
- Adds Airlock and EVA access to explorers
- Increases number of pilot slots by 2
- Adds CMO access to the medical shuttle
NEW:
- Adds Command channel to Quartermaster job role
- Moves Exploration to under the HoP
- Removes HoP from showing up under Cargo (having the HoP show up for every subdepartment they lead is silly)

SOP notes:
- HoP will be in charge of managing Exploration personnel (Firing, hiring, etc)
- Field Medic will remain as a hybrid Medical/Explo role under PF/CMO authority
- Research Directors will no longer be exempted from the general rule that heads of staff should not attend expeditions.
- Exploration is still strongly encouraged to get volunteers from other departments, including Science.

### CODE REVIEW

File: vorestation.dme [hunk 1]
```diff
@@ -1617,10 +1619,10 @@
...
 #include "code\modules\ai\interfaces.dm"
 #include "code\modules\ai\say_list.dm"
-#include "code\modules\ai\ai_holder_subtypes\simple_mob_ai.dm"
-#include "code\modules\ai\ai_holder_subtypes\simple_mob_ai_vr.dm"
-#include "code\modules\ai\ai_holder_subtypes\slime_xenobio_ai.dm"
-#include "code\modules\ai\ai_holder_subtypes\slime_xenobio_ai_vr.dm"
+#include "code\modules\ai\aI_holder_subtypes\simple_mob_ai.dm"
+#include "code\modules\ai\aI_holder_subtypes\simple_mob_ai_vr.dm"
+#include "code\modules\ai\aI_holder_subtypes\slime_xenobio_ai.dm"
+#include "code\modules\ai\aI_holder_subtypes\slime_xenobio_ai_vr.dm"
```
Comment (Leshana) [inline vorestation.dme:1625]: Are you a goose?

Comment (Novacat) [reply vorestation.dme:1625]: No, oddly enough Dreammaker did that automatically. I have no idea why.

File: code/game/jobs/job/exploration_vr.dm [hunk 2]
```diff
@@ -39,16 +39,16 @@ var/const/SAR 				=(1<<14)
 	faction = "Station"
 	total_positions = 1
 	spawn_positions = 1
-	supervisors = "the research director"
+	supervisors = "the Colony Director"
 	selection_color = "#d6d05c"
-	economic_modifier = 7
+	economic_modifier = 8
```
Comment (KasparoVy) [inline code/game/jobs/job/exploration_vr.dm:44]: Hazard pay!

**Comment by Arokha** [comment]
>Pathfinder will be able to fire subordinates.
>Pathfinder will not be a full head.

I mean what's the difference between them and a full head in terms of policies at that point?

**Comment by Arokha** [comment]
Is the SOP updated?

**Comment by Novacat** [comment]
> I mean what's the difference between them and a full head in terms of policies at that point? They even report to the CD like other heads, and not to a 'real head' like our existing fake-head, the QM.

- Pathfinder is not in the chain of command to be acting head.
- Pathfinder does not have head access, nor access to the bridge (though they will probably gain this access indirectly in Aubergine due to pilots needing bridge access to fly the ship)
- Pathfinder is not required to participate in any meetings that require a gathering of heads, such as Tribunals.

The TL;DR is that Pathfinders, unlike most heads, do not make decisions that would affect the whole station. That is the main thing that separates them from being a full head. Do note that the only major functional changes being done are that PF can fire expedition staff, gets access to command channel, and reports directly to the CD instead of the RD.

**Comment by Arokha** [comment]
That's a pretty weird state IMO. They have all the abilities of a head without any of the responsibilities? They should probably report to the HoP and not have command channel access and not be able to fire staff.

It shouldn't be in the middle. It should be a 'fake head' like QM, or a real head like every other head, not a mix of the two. Don't create more weird halfway positions that aren't even similar to our weird halfway positions already.

**Pull request merged**